corehq/apps/hqadmin/views.py

```diff
@@ -440,6 +447,18 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class DomainAdminRestoreView(AdminRestoreView):
+    urlname = 'domain_admin_restore'
+
+    def dispatch(self, request, *args, **kwargs):
+        return TemplateView.dispatch(self, request, *args, **kwargs)
```

Comment:
What if you made this the base class and `AdminRestoreView` was the one that inherited? would probably want to rename the classes to `SuperuserAdminRestoreView`